content/docs/command-reference/checkout.md

```diff
@@ -10,21 +10,26 @@ usage: dvc checkout [-h] [-q | -v] [--summary] [-d] [-R] [-f]
                     [--relink] [targets [targets ...]]
 
 positional arguments:
-  targets        Limit command scope to these stages or .dvc files.
-                 Using -R, directories to search for stages or .dvc
-                 files can also be given.
+  targets       Limit command scope to these tracked files/directories,
+                .dvc files, or stage names.
 ```
 
 ## Description
 
-`.dvc` and `dvc.lock` [files](/doc/user-guide/dvc-files-and-directories) act as
-pointers to specific version of data files or directories tracked by DVC. This
-command synchronizes the workspace data with the versions specified in the
-current `.dvc` and `dvc.lock` files.
+Synchronizes the workspace data with all the
+[tracked](/doc/command-reference/add) files and directories specified in the
+present `dvc.lock` and `.dvc` files, which act as pointers to corresponding
+<abbr>cached</abbr> data contents.
+
+The `targets` given to this command (if any) limit which files to checkout. It
+accepts paths to tracked files or directories (even if such paths are within a
+directory
+[tracked as a whole](/doc/command-reference/add#tracking-directories)), `.dvc`
+files, or stage names (found in `dvc.yaml`).
 
 `dvc checkout` is useful, for example, when using Git in the
 <abbr>project</abbr>, after `git clone`, `git checkout`, or any other operation
-that changes the DVC files in the workspace.
+that changes the DVC files in the <abbr>workspace</abbr>.
```

Comment:
this is not the first mention of the workspace, btw

Reply:
The first one is in line 3 and has abbr. The next one in line 20 probably is shown in the same "page" (without scrolling) so I skipped abbring that one. That's my general rule although I'm sure there are cases where this is not consistent...